src/modules/Thread/components/ThreadUI/index.tsx

```diff
@@ -148,7 +148,7 @@ const ThreadUI: React.FC<ThreadUIProps> = ({
         )
       }
       <div
-        className="sendbird-thread-ui--scroll"
+        className={'sendbird-thread-ui--scroll sendbird-conversation__messages'}
```

Comment:
let's use the `classnames` function
```tsx
className={classnames('sendbird-thread-ui--scroll', 'sendbird-conversation__messages')}
```

Reply:
This one too https://github.com/sendbird/sendbird-uikit-react/pull/1184/commits/52397fd40e6d71fb54851ed276da8c1d950cafa3